### PR TITLE
Feat(Barcode in POS) using barcodes in stock receipts

### LINKED ISSUE #4123: Add barcodes for the stock document_uuid to stock POS receipts
We should have barcodes for the stock thermal receipts.  We should also remove the cost values from these receipts since we removed them from the main A4 receipts.

### PR DESCRIPTION
**Pull request by jeremielodi**
closes #4123

barcode-receipt

### CODE REVIEW

File: server/controllers/stock/reports/stock/exit_patient_receipt.js [hunk 1]
```diff
@@ -22,6 +22,10 @@ async function stockExitPatientReceipt(documentUuid, session, options) {
   if (Boolean(Number(optionReport.posReceipt))) {
     template = POS_STOCK_EXIT_PATIENT_TEMPLATE;
     _.extend(optionReport, pdf.posReceiptOptions);
+    // provide barcode string to be rendered by client/ receipts
+    const entityIdentifier = identifiers.STOCK_EXIT.key;
+    const barcodeString = barcode.generate(entityIdentifier, documentUuid);
+    data.barcode = barcodeString;
```
Comment (jniles) [inline server/controllers/stock/reports/stock/exit_patient_receipt.js:28]: 👍

File: server/lib/template/partials/barcode-header.handlebars [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<head>
+  <meta charset="utf-8" />
+  <script src="{{nodeModulesPath}}/jsbarcode/dist/JsBarcode.all.min.js"></script>
```
Comment (jniles) [inline server/lib/template/partials/barcode-header.handlebars:3]: 👍

File: server/controllers/stock/reports/stock_exit_service.receipt.pos.handlebars [hunk 3]
```diff
@@ -1,6 +1,6 @@
-<head>
-  <meta charset="utf-8" />
-</head>
+
+{{> barcode-header title="STOCK.RECEIPT.EXIT_SERVICE" }}
```
Comment (jniles) [inline server/controllers/stock/reports/stock_exit_service.receipt.pos.handlebars]: It doesn't look like you use the `title` property.  Could you remove it?

File: server/controllers/stock/reports/stock_exit_depot.receipt.pos.handlebars [hunk 4]
```diff
@@ -72,5 +72,5 @@
     <hr>
   </div>
 </footer>
-
+ <script>JsBarcode('.barcode').init();</script>
```
Comment (jniles) [inline server/controllers/stock/reports/stock_exit_depot.receipt.pos.handlebars]: It looks like this should have the 
```hbs
 {{#if metadata.enterprise.settings.enable_barcodes}}
```

File: server/controllers/stock/reports/stock_exit_patient.receipt.pos.handlebars [hunk 5]
```diff
@@ -1,6 +1,4 @@
-<head>
-  <meta charset="utf-8" />
-</head>
+{{> barcode-header title="STOCK.RECEIPT.EXIT_PATIENT" }}
```
Comment (jniles) [inline server/controllers/stock/reports/stock_exit_patient.receipt.pos.handlebars:1]: Same comment about the `title`.

**Review by jniles — COMMENTED** [review]
@jeremielodi this looks good!

Just a few minor comments here.

**Review by mbayopanda — COMMENTED** [review]
The barcode works well 👍 

I have just noticed that for stock module which has POS receipts only `Stock Exit to Patient` and `Stock Exit to Service` have barcode on their receipts.

These receipts below doesn't have barcodes : 
- Stock exit to a depot
- Stock exit to a loss

**Comment by mbayopanda** [comment]
@jeremielodi have you done with this PR ?

**Comment by jeremielodi** [comment]
I'm working on it today and come back to you for a review

**Pull request merged**